flexget/plugins/sites/btn.py

```diff
@@ -47,14 +62,15 @@ def search(self, task, entry, config):
                     search['name'] = entry['series_id'] + '%'  # added wildcard search for better results.
             searches = [search]
             # If searching by series name ending in a parenthetical, try again without it if there are no results.
-            if search.get('series') and search['series'].endswith(')'):
+            if search.get('series') and search['series'].endswith(')') and not config.get('exact_only'):
                 match = re.match('(.+)\([^\(\)]+\)$', search['series'])
                 if match:
                     searches.append(dict(search, series=match.group(1).strip()))
 
+        log.debug(searches)
         results = set()
         for search in searches:
-            data = json.dumps({'method': 'getTorrents', 'params': [api_key, search], 'id': 1})
+            data = json.dumps({'method': 'getTorrents', 'params': [config.get('api_key'), search], 'id': 1})
```

Comment:
Minor detail, `api_key` is always present, so there's no need to use `config.get('api_key')` over `config[api_key]`. I prefer to use the latter in that scenario.